deploy-pgsql.sh

```diff
@@ -15,7 +15,6 @@ docker run --detach \
     --name=pgsql \
     --network=sourcegraph \
     --restart=always \
-    --cpus=4 \
```

Comment:
why was this removed? :o

Reply:
I was running the deploy scripts locally and kept seeing some CPU issues, this should not be here